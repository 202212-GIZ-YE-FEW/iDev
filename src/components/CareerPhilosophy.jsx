import { withTranslation } from "next-i18next";

import Button from "./ui/Button";

const CareerPhilosophy = ({ t }) => {
    return (
        <div>
            <div className='lg:text-xl text-base leading-6 lg:mt-24 mt-16 lg:ms-28 ms-10 lg:me-52 md:me-20 me-10 text-[#696f72]'>
                {t("ourSeoSoftware")}
            </div>

            <div className='lg:text-2xl text-lg leading-6 mt-20 lg:ms-28 ms-10 text-[#696f72]'>
                <h3>{t("whatRole")}</h3>
            </div>

            <div className=' mt-4 lg:ms-28 mb-8 ms-10 '>
                <Button
                    content={t("SeeOurOpenList")}
                    textTransform='uppercase'
                    filled='true'
                    size='large'
                    fontSize='text-lg md:text-xl lg:text-2xl'
                    radius='md'
                />
            </div>

            <div className='text-gray bg-yellow pt-4 pb-4'>
                <div className='lg:text-5xl md:text-3xl text-2xl leading-6 mt-10 lg:ms-28 ms-10'>
                    <h1>{t("ourHiringPhi")}</h1>
                </div>
                <div className='lg:text-lg text-base leading-6 mt-6 lg:ms-28 ms-10 lg:me-52 md:me-20 me-10 mb-8'>
                    {t("toBuildTheVeryBestSeo")}
                </div>
            </div>
        </div>
    );
};

export default withTranslation("career")(CareerPhilosophy);
