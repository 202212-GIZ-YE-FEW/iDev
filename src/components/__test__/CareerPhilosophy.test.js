import renderer from "react-test-renderer";

import CareerPhilosophy from "../CareerPhilosophy";

it("renders correctly", () => {
    const tree = renderer.create(<CareerPhilosophy />).toJSON();
    expect(tree).toMatchSnapshot();
});
