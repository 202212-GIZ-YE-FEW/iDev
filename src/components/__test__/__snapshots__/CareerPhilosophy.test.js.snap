// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <div
    className="lg:text-xl text-base leading-6 lg:mt-24 mt-16 lg:ms-28 ms-10 lg:me-52 md:me-20 me-10 text-[#696f72]"
  >
    ourSeoSoftware
  </div>
  <div
    className="lg:text-2xl text-lg leading-6 mt-20 lg:ms-28 ms-10 text-[#696f72]"
  >
    <h3>
      whatRole
    </h3>
  </div>
  <div
    className=" mt-4 lg:ms-28 mb-8 ms-10 "
  >
    <div
      className="relative self-center"
    >
      <button
        className="transform hover:bg-yellow transition duration-500  hover:animate-bounce hover:scale-75 active:bg-cyan focus:outline-none focus:ring focus:ring-cyan
                font-normal whitespace-nowrap cursor-pointer  uppercase text-lg md:text-xl lg:text-2xl rounded-md px-[25px] md:px-[35px] py-[8px] font-medium bg-cyan text-black "
        disabled={false}
        onClick={[Function]}
      >
        SeeOurOpenList
      </button>
    </div>
  </div>
  <div
    className="text-[#696f72] bg-yellow"
  >
    <div
      className="lg:text-5xl md:text-3xl text-2xl leading-6 mt-10 lg:ms-28 ms-10"
    >
      <h1>
        ourHiringPhi
      </h1>
    </div>
    <div
      className="lg:text-lg text-base leading-6 mt-6 lg:ms-28 ms-10 lg:me-52 md:me-20 me-10 mb-8"
    >
      toBuildTheVeryBestSeo
    </div>
  </div>
</div>
`;
